cg/models/cg_config.py

```diff
@@ -206,7 +206,7 @@ class GisaidConfig(CommonAppConfig):
 class DataDeliveryConfig(BaseModel):
     destination_path: str
     covid_destination_path: str
-    covid_source_path = str
```

Comment:
I assume this was a typo. The field had to be marked as optional since tests started failing after just correcting the type hint

Reply:
I would not be so sure about it - I would look further into the data delivery of sars-cov-2 to make sure that it is in fact correct

Reply:
I'm 100 % it is a typo, type hints are assigned with `: str` not `= str` and it makes no sense to assign initialise it as `str`. But it might be the case that `covid_source_path` should not be optional, but that is in fact the case previously.

Reply:
If it should not be optional, it is new behaviour. Creating an issue for it and we can refine it in a refinement session.

Reply:
https://github.com/Clinical-Genomics/cg/issues/2415